docs/metadata/prophecy-build-tool.md

```diff
@@ -96,12 +107,35 @@ Querying existing jobs to find current job: Offset: 0, Pagesize: 25
 ✅ Deployment completed successfully!
 ```
 
+The `deploy` command also supports an advanced option `--dependent-projects-path` if there is a need to build projects other than the main project that has to be deployed.
```

Comment:
Does this support multiple paths? Can we see an example of this?

Reply:
Added more details